package.json

```diff
@@ -8,16 +8,16 @@
   },
   "bugs": "https://github.com/TryGhost/Ghost/issues",
   "private": true,
-  "version": "0.11.3",
+  "version": "0.11.8",
   "dependencies": {
-    "casper": "TryGhost/Casper#1.3.4",
-    "ghost": "0.11.3",
+    "casper": "TryGhost/Casper#1.3.6",
+    "ghost": "0.11.8",
     "ghost-s3-storage-adapter": "3.0.4",
     "ncp": "^2.0.0",
     "pg": "latest"
   },
   "engines": {
-    "node": "~0.12.0"
+    "node": "^6.10.3"
```

Comment:
Yeah, I think it helps to establish more patterns for future decisions regarding this repo too. The more we can defer to outside influences (i.e. upstream ghost) for decisions, the simpler it is for anyone to update things like this going forward. 👍